core/src/main/scala/com/rallyhealth/vapors/factfilter/dsl/Optimizer.scala

```diff
@@ -0,0 +1,14 @@
+package com.rallyhealth.vapors.factfilter.dsl
+
+// TODO: Replace this with explicit sorting within expressions
+trait Optimizer[F[_], V] {
+
+  def optimize(values: F[V]): F[V]
+}
```

Comment:
I'm not sure if I should use this for sorting the facts in the `FactTable` or if I should just remove it. In general, I think it is better to sort values using a sort `Expr` node, rather than creating a one-off typeclass like this. The only exception, of course, would be to update the `FactTable` I suppose.

Reply:
Removed.